src/jest/users.test.ts

```diff
@@ -76,3 +68,91 @@ describe('FIREBASE User tests', () => {
     await expect(getUserById(testUser1.id!)).rejects.toThrow(BaseError);
   });
 });
+
+/******************/
+/**** Express  ****/
+/******************/
+
+describe('EXPRESS user routes', () => {
+  let expressId: string;
+  let expressToken: string;
+  let hashedPassword: string;
+  const orgid = 1232344432 as number;
+  const user1: { [key: string]: any } = {
+    firstName: 'Thor',
+    lastName: 'Hansen',
+    email: 'express@mail.com',
+    password: 'Test123!!!',
+  };
+  test('Create user', async () => {
+    const res = await request(app)
+      .post('/users/' + orgid)
+      .send(user1);
+
+    expressId = res.body.id;
+    expressToken = res.body.token;
+    hashedPassword = res.body.password;
+    expect(res.statusCode).toBe(201);
+    expect(res.body.firstName).toBe(user1.firstName);
+    expect(res.body.lastName).toBe(user1.lastName);
+    expect(res.body.email).toBe(user1.email);
+    expect(res.body.password).toBeDefined();
```

Comment:
Passwords should NEVER be returned by the API. Not hashed or plain.
```suggestion
    expect(res.body).not.toHaveProperty('password');
```